pom.xml

```
@@ -62,7 +62,7 @@
     <zipkin-proto3.version>1.0.0</zipkin-proto3.version>
 
     <!-- Do not set this value in bom/pom.xml to avoid cyclic pinning -->
-    <brave.version>5.13.2</brave.version>
+    <brave.version>5.15.1</brave.version>
```

thanks to you there are more versions ;)

```suggestion
    <brave.version>5.16.0</brave.version>
```